src/App.js

```diff
@@ -1,7 +1,7 @@
 import { render } from '@testing-library/react';
 import React from 'react';
-
-import ToDo from './todo/todo.js';
+import './todo/todo.scss';
+import ToDo from './todo/todo-connected.js';
 
 export default function App(props) {
     return (
```

Comment:
Code is looking great so far, I would like to see axios added as a dependency and being used